src/content/docs/iast/install.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Install New Relic IAST
+tags:
+    - New Relic IAST
+    - IAST
+    - Installation
+metaDescription: Learn how to install New Relic IAST.
+---
+
+import iastInstall from 'images/iast_screenshot-crop_install.webp'
+
+<Callout variant="important">
+  Run IAST with non-production deployments only to avoid exposing vulnerabilities on your production software.
+
+  IAST tests your applications for any exploitable vulnerability by replaying the generated HTTP request with vulnerable payloads.
+</Callout>
+
+To install New Relic IAST:
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST**, and click **Install** on the left navigation pane.
+2. In the installation window, select the language of your application and complete the steps.
+    <img
+        title="Install New Relic IAST"
+        alt="Install New Relic IAST"
+        src={iastInstall}
+    />
+
+    <figcaption>
+        Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST > Install**, select the language of your application, and complete the steps.
+    </figcaption>
+
+3. After you've installed your application, use the APIs from your application so that New Relic can start looking for exploitable vulnerabilities. You can do so by running your own tests against your APIs.
```

Comment:
I worry that this step 3 isn't detailed enough. What if I don't know how to do these things with APIs? Also, we try not to include more than one sentence to a step.